README.md

```diff
@@ -48,11 +48,11 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 [Kubernetes Code of Conduct]: code-of-conduct.md
 [community page]: https://kubernetes.io/community/
-[contributor guide]: https://sigs.k8s.io/hydrophone/blob/main/CONTRIBUTING.md
+[contributor guide]: https://github.com/kubernetes-sigs/hydrophone/blob/main/CONTRIBUTING.md
```

Comment:
[minor] @rakshitgondwal, I wonder if we should just put the contrib guide filename instead of this URL? That'll work better with forks and local tests.

Reply:
Good point, let's do that @UtkarshUmre